ee/tabby-ui/app/(dashboard)/settings/(integrations)/mail/components/mail-testing-form.tsx

```diff
@@ -17,36 +20,50 @@ import {
 import { Input } from '@/components/ui/input'
 import { Label } from '@/components/ui/label'
 
+const sendTestEmailMutation = graphql(/* GraphQL */ `
+  mutation SendTestEmail($to: String!) {
+    sendTestEmail(to: $to)
+  }
+`)
+
 const formSchema = z.object({
-  email: z.string().email('Invalid email address')
+  to: z.string().email('Invalid email address')
 })
 
+type FormValues = z.infer<typeof formSchema>
+
 export default function MailTestingForm({
   onSendTest
 }: {
   onSendTest: () => Promise<any>
 }) {
-  const form = useForm<z.infer<typeof formSchema>>({
+  const form = useForm<FormValues>({
     resolver: zodResolver(formSchema)
   })
 
   const { isSubmitting } = form.formState
-  const handleSubmitTest = () => {
-    onSendTest()
+  const sendTestEmail = useMutation(sendTestEmailMutation, { form })
+  const onSubmit = (values: FormValues) => {
+    sendTestEmail(values).then(res => {
+      if (res?.data?.sendTestEmail) {
+        toast.success('email sent')
```

Comment:
```suggestion
        toast.success('A test email has been sent, please check your inbox to verify.')
```

Please pay more attention to all user facing text. They're part of the user experience.